app/models/project.rb

```diff
@@ -38,17 +46,22 @@ class Project < ApplicationRecord
            source: :user
 
   validates :github_id, presence: true, uniqueness: true
+  validates :relevance, inclusion: { in: relevances.keys }
 
   before_validation :set_default_language, on: :create
 
   scope :open_source, lambda {
-    with_language.where(is_private: false)
+    internal.with_language.where(is_private: false)
```

Comment:
maybe a spec to show that open source projects don't include `ignored` repos

Reply:
Actually, [this spec](https://github.com/rootstrap/rs-code-review-metrics/pull/278/files#diff-fb312d8dcf15fea2e04d19523d09ff27R74) is already doing that (although it's an easy miss, I must admit 😅 )
I create several projects in the setup with different attributes, and one of them is precisely an `ignored_project`, which I'm asserting it's not showing up as an open source project (although I'm not doing it explicitly).
If you think it's a bit obscure, I could create different contexts and explicitly assert the different type of projects are not being included

Reply:
I missed that one, I'm good then 👌